app/controllers/oligrapher_controller.rb

```diff
@@ -94,6 +94,27 @@ def find_nodes
     render json: entities
   end
 
+  def find_connections
+    return head :bad_request if params[:entity_id].blank?
+
+    @entity = Entity.unscoped.find(params[:entity_id])
+    raise Entity::EntityDeleted if @entity.is_deleted?
+
+    entities = EntityConnectionsQuery
+      .new(@entity)
+      .category(params[:category_id])
+      .page(1)
+      .per(params.fetch(:num, 10))
+      .run
+      .map { |e|
```

Comment:
A future reactor of `Oligrapher` could simplify these lines (this note is probably mostly for me but...)

It might be nice to have a method on `Oligrapher` with this functionality:  `node_with_edge`

Then the chain becomes more readable readable: ` .map { |e| Oligarpher.node_with_edge(e) } `